requirements/prod.txt

```diff
@@ -38,7 +38,7 @@ django-statsd-mozilla==0.4.0
 django-two-factor-auth==1.9.1
 django-webpack-loader==0.6.0
 django==1.11.24
-# future==0.17.1            # via python-twitter
+# future==0.18.2            # via python-twitter
```

Comment:
I think this probably happened because of some rebasing issue. If rebasing doesn't fix this, try `git reset HEAD^` and then unstage this before committing.

Reply:
@vinitS101 resetting worked. No errors while pushing this time.